dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/event/RumEventMapper.kt

```diff
@@ -71,6 +84,21 @@ internal data class RumEventMapper(
         }
     }
 
+    private fun notifyEventDropped(event: RumEvent) {
+        val monitor = (GlobalRum.get() as? AdvancedRumMonitor) ?: return
+        when (val rumEvent = event.event) {
+            is ActionEvent -> monitor.eventDropped(rumEvent.view.id, EventType.ACTION)
+            is ResourceEvent -> monitor.eventDropped(rumEvent.view.id, EventType.RESOURCE)
+            is ErrorEvent -> monitor.eventDropped(rumEvent.view.id, EventType.ERROR)
+            is LongTaskEvent -> monitor.eventDropped(rumEvent.view.id, EventType.LONG_TASK)
+            else -> {
+                // Nothing to do
```

Comment:
shouldn't we print some warning here? will be helpful in case if new event type is added to the codebase, but contributor forgot to handle it here.

on another side I see that `event.event` type is `Any`, so curious how often this branch will be hit.